src/Umbraco.Infrastructure/Persistence/Repositories/Implement/TemplateRepository.cs

```diff
@@ -449,9 +458,28 @@ public Stream GetFileContentStream(string filepath)
             }
         }
 
+        public void SetFileContent(string filepath, Stream content) => GetFileSystem(filepath).AddFile(filepath, content, true);
+
+        public long GetFileSize(string filename)
+        {
+            if (GetFileSystem(filename).FileExists(filename) == false)
+            {
+                return -1;
+            }
+
+            try
+            {
+                return GetFileSystem(filename).GetSize(filename);
```

Comment:
Small thing, but is there a reason to call GetFileSystem twice? Instead of saving it in a variable once, and using that twice?

Reply:
No, this was just a copy and paste, but I'll amend, as you're right, there's no reason.